src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/PublicConfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "PublicConfig": { "UserName": "dileep" }
```

Comment:
This looks like a real user name, can we substitue a more obviously fake user name?.